### PR TITLE
Proposal for contribution guidelines

### PR DESCRIPTION
**Pull request by tsadler1988**
📺 What

Add contribution guidelines

> Tickets: N/A

🛠 How

CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We welcome and encourage contributions. We are open minded to suggestions, encourage debate, and aim to build a consensus; if there is no consensus, maintainers will have the final say.
```
Comment (matt-stephenson) [inline CONTRIBUTING.md:3]: Worth saying who the 'we' is in this first usage?

Comment (matt-stephenson) [inline CONTRIBUTING.md:3]: I read the 'maintainers will have the final say' and felt it was a bit 'strongly worded' for the opening sentence. Could we:
a) make it less so
b) just not write this, as it's essentially covered below in the review/test/release process (maybe make that section more explicit to explain the point)

Comment (tsadler1988) [reply CONTRIBUTING.md:3]: I'd have thought that was fairly clear. Not sure what else we'd put:
- BBC is too generic
- Honey Badger only makes sense internally
- Bigscreen Player maintainers is a bit wordy

Comment (tsadler1988) [reply CONTRIBUTING.md:3]: Yes I wasn't sure about putting this in. Perhaps aimed more at internal contributors - I think it's fairly accepted that maintainers have the final say in Open Source. Best to just remove it?

Comment (matt-stephenson) [reply CONTRIBUTING.md:3]: Yep, fair point 👍

Comment (matt-stephenson) [reply CONTRIBUTING.md:3]: 👍
